playwright.config.ts

```diff
@@ -0,0 +1,13 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'npm run build && npm run preview',
+    port: 4173,
+  },
+  testDir: 'tests',
+  // eslint-disable-next-line sonarjs/slow-regex
+  testMatch: /(.+\.)?(test|spec)\.[jt]s/,
+};
+
+export default config;
```

Comment:
we do not have playwright test in this ?

Reply:
I'll remove the file generated by sveltekit